database/handler/process_handler.py

```diff
@@ -0,0 +1,244 @@
+    """
+
+    data = {
+        "success": True,
+        "process": [
+            {
+                "uid": "b141f94973a43cf8ee972e9dffc1b004",
+                "name": "Kunde anlegen",
+                "creation_timestamp": "20210210...",
+                "last_timestamp": "20200211..."
+            },
+            {
+                "uid": "b141f94973a43cf8ee972e9dffc1b004",
+                "name": "Kunde löschen",
+                "creation_timestamp": "20210209...",
+                "last_timestamp": "20210210..."
+            }
+        ]
+    }
+
+    return data
+
+
+def get_process(uid_dict: dict) -> dict:
+    """
+    Function to retrieve a single process
+
+    :param uid_dict: process uid
+    :type uid_dict: dict
+    :return: process dict
+    """
+
+    data = {
+        "success": True,
+        "process": {
+            "uid": "b141f94973a43cf8ee972e9dffc1b004",
+            "name": "Kunde anlegen",
+            "description": "Prozess zum anlegen von einem neuen Kunden in allen Systemen",
+            "creation_timestamp": "20210210...",
+            "last_timestamp": "20200211...",
+            "components": [
+                {
+                    "uid": "b141f94973a43cf8ee972e9dffc1b004",
+                    "weight": 1,  # different from single component view!
+                    "name": "SQL Datenbank",
+                    "category": "Datenbank",
+                    "description": "Kundendatenbank",
+                    "creation_timestamp": "20200219...",
+                    "last_timestamp": "20200219...",
+                    "metrics": {
+                        "codelines": 20000,
+                        "admins": 10,
+                        "recovery_time": 5
+                    }
+                },
+                {
+                    "uid": "b141f94973a43cf8ee972e9dffc1b004",
+                    "weight": 1.5,
+                    "name": "Frontend API",
+                    "category": "API",
+                    "description": "API für das Frontend",
+                    "creation_timestamp": "20200219...",
+                    "last_timestamp": "20200219...",
+                    "metrics": {
+                        "codelines": 20000,
+                        "admins": 10,
+                        "recovery_time": 5
+                    }
+                },
+                {
+                    "uid": "b141f94973a43cf8ee972e9dffc1b004",
+                    "weight": 2,
+                    "name": "Hadoop Cluster",
+                    "category": "Datenbank",
+                    "description": "Big Data Plattform",
+                    "creation_timestamp": "20200219...",
+                    "last_timestamp": "20200219...",
+                    "metrics": {
+                        "codelines": 20000,
+                        "admins": 10,
+                        "recovery_time": 5
+                    }
+                }
+            ]
+        },
+        "should-metrics": {
```

Comment:
```suggestion
        "target_metrics": {
```

Haben wir glaube ich umbennant